vizro-ai/src/vizro_ai/_vizro_ai.py

```diff
@@ -136,75 +131,3 @@ def plot(
         # TODO Tentative for integration test
         if self._return_all_text:
             return output_dict
-
-
-def _debug_helper(
-    code_string: str, max_debug_retry: int, fix_chain: Callable, df: pd.DataFrame = None
-) -> Dict[bool, str]:
-    """Debugging helper."""
-    # TODO plug logic back into component
-    retry_success = False
-    last_exception = None
-    for attempt in range(max_debug_retry):
-        try:
-            _exec_code(code=code_string, local_args={"df": df}, is_notebook_env=_is_jupyter())
-            retry_success = True
-            break
-        except Exception as e:
-            error_info = f"{traceback.format_exc()}"
-            code_string = fix_chain(chain_input=error_info, code_snippet=code_string)
-            last_exception = e
-
-    code_string = code_string if retry_success else f"Failed to debug code {code_string}, error: {last_exception}"
-
-    return {"debug_status": retry_success, "code_string": code_string}
-
-
-def _exec_code(
-    code: str, local_args: Optional[Dict] = None, show_fig: bool = False, is_notebook_env: bool = True
-) -> None:
-    """Execute code in notebook with correct namespace."""
-    from IPython import get_ipython
-
-    if show_fig and "\nfig.show()" not in code:
-        code += "\nfig.show()"
-    elif not show_fig:
```

Comment:
Can this be converted to a simple else? If not is there an else missing? It can also be a simple `else: pass` just for readibility?